### PR TITLE
Providing Typedoc references for the Grid GroupBy topic.

### PR DESCRIPTION
**Pull request by MartinKamenov**

### CODE REVIEW

**Comment by yoshimis** [comment]
Reviewed and no localization necessary for this PR. Please add a label "pending-localization" back if any new changes added.

File: jp/components/grid_groupby.md [hunk 1]
```diff
@@ -127,6 +127,13 @@ export interface IGroupByRecord {
 |--- |--- |
 |グループ列の最大値は 10 です。 | 10 列以上の場合はエラーがスローされます。
 
+### API
+
+* [IgxGridComponent]({environment:angularApiUrl}/classes/igxgridcomponent.html)
```
Comment (HristoP96) [inline jp/components/grid_groupby.md]: Provide Sassdoc reference for the igx-grid-theme

File: jp/components/grid_groupby.md [hunk 2]
```diff
@@ -127,6 +127,13 @@ export interface IGroupByRecord {
...
+* [ISortingExpression]({environment:angularApiUrl}/interfaces/isortingexpression.html)
+* [IgxColumnComponent]({environment:angularApiUrl}/classes/igxcolumncomponent.html)
+* [IGroupByExpandState]({environment:angularApiUrl}/interfaces/igroupbyexpandstate.html)
+
```
Comment (HristoP96) [inline jp/components/grid_groupby.md]: Could provide Typedoc and Sassdoc references for IgxChipCompoenent and igx-chip-theme, respecrively

Comment (MartinKamenov) [reply jp/components/grid_groupby.md]: @HristoP96 Please check the changes that were made.

**Pull request merged**